### PR TITLE
44 mock new data source

### PR DESCRIPTION
**Pull request by Kpaubert**
Mocking of the SSB-API giving out population per municipality over several years. Also refactored the names of classes named "AgesIn..." to "PopulationIn...".

Part of #44

### CODE REVIEW

File: Mocks/SSB/MockPopulationInNorway.cs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        [SuppressMessage("Rule Category", "CA5394", Justification = "No security threat on data mocking.")]
+        public static PopulationPerMunicipalityNorway GenerateSampleAgesInNorway()
+        {
+            Random random = new();
+            var numYears = NorwayTools.YearToIndex.Count;
+            var numMunicipalities = NorwayTools.MunicipalityCodeToIndex.Count;
+            const int approxPopulationNorway = 5391369;
+
+            var region = new Faker<JsonStatDimensionContent>()
+                .RuleFor(o => o.Category, f => new JsonStatDimensionCategory()
```
Comment (fredrikbw) [inline Mocks/SSB/MockPopulationInNorway.cs:30]: Replace f with _

File: Mocks/SSB/MockPopulationInNorway.cs [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                    Label = NorwayTools.MunicipalityCodeToMunicipalityName
+                })
+                .RuleFor(o => o.Label, _ => "Label");
+            var time = new Faker<JsonStatDimensionContent>()
+                .RuleFor(o => o.Category, _ => new JsonStatDimensionCategory()
+                {
+                    Index = NorwayTools.YearToIndex,
+                    Label = NorwayTools.YearToYearString
+                })
+                .RuleFor(o => o.Label, f => "År");
```
Comment (fredrikbw) [inline Mocks/SSB/MockPopulationInNorway.cs:42]: ^

**Review by fredrikbw — CHANGES_REQUESTED** [review]
Very good work, just some nitpicking

**Review by fredrikbw — APPROVED** [review]
LGTM

File: Mocks/SSB/MockPopulationInNorway.cs [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    /// by municipality, using the JSON-stat format. The municipalities
+    /// and their codes are hardcoded, and the population rates ascending
+    /// in a pseudorandom fashion.
+    /// </summary>
+    public sealed class MockPopulationInNorway
+    {
+        [SuppressMessage("Rule Category", "CA5394", Justification = "No security threat on data mocking.")]
+        public static PopulationPerMunicipalityNorway GenerateSampleAgesInNorway()
+        {
+            Random random = new();
```
Comment (FredrikAugust) [inline Mocks/SSB/MockPopulationInNorway.cs:24]: use built-in instead

**Pull request merged**